app/lib/audit/models.dart

```diff
@@ -209,10 +209,14 @@ class AuditLogRecord extends db.ExpandoModel<String> {
     required DateTime created,
     String? publisherId,
   }) {
+    final uploaderFields = uploader.fields;
+    final formattedFields = uploaderFields == null
+        ? ''
+        : ' (${uploaderFields.entries.map((e) => '${e.key}: `${e.value}`').join(', ')})';
     final summary = [
       'Package `$package` version `$version`',
       if (publisherId != null) ' owned by publisher `$publisherId`',
-      ' was published by `${uploader.displayId}`.',
+      ' was published by `${uploader.displayId}`$formattedFields.',
```

Comment:
This is a user-facing message, and security it's probably relatively important that users understand this message.
Might I suggest something along the lines of:
```markdown
Package `retry` version `1.2.3` was published from Github Actions running on revision `a2a2a2aa2a`
from the `google/dart-neats` repository.
The authenticating Github Actions flow was triggered by a user identified as `jonasfj` on Github (at the time of publishing).
```

Reply:
Updated. PTAL.